optaweb-vehicle-routing-docs/src/main/asciidoc/introduction.adoc

```diff
@@ -0,0 +1,15 @@
+== Introduction
+
+=== What is OptaWeb Vehicle Routing?
+
+The main purpose of many types of businesses is to transport various types of cargo.
+The goal of these businesses is to deliver a piece of cargo from the loading point to a destination and use its vehicle fleet in the most efficient way.
```

Comment:
I suggest that you consider using the text that I rewrote in our review.

Reply:
Yes, that was my intention. Let me check where I did a mistake...

Reply:
I used the text from #269. Is there another, more up-to-date version, that I may have missed?